src/components/CreateVmWizard/steps/style.css

```diff
@@ -111,6 +111,10 @@
   display: flex;
   align-items: center;
 }
+.disk-size-form-control-edit {
+  width: 0;
```

Comment:
Last question.

Does the layout work with `width: auto` instead of `width: 0`?  The `auto` value is a bit more typical to see when you want to reset a style that gets inherited.

Reply:
it works but using `width:auto` stretches the whole column and I thought it will be better if the table structure doesn't change between modes. 
That how it looks on edit mode:
![image](https://user-images.githubusercontent.com/64131213/83634313-50476d80-a570-11ea-8d10-f92eafc9d4e6.png)
That how it looks on view mode:
![image](https://user-images.githubusercontent.com/64131213/83634959-6b66ad00-a571-11ea-9ba5-ac5a40fe6e54.png)